WordGuessGame.java

```diff
@@ -31,6 +36,26 @@ public Optional<Integer> play() {
             "Your score is determined by the number of attempts remaining"
             + " after you guessed the word correctly!"
         );
-        return Optional.empty();
+		
+		String guess = EnterGuess();
+		guess = GuessData(guess);
+		NumberOfGuesses--;
+		return Optional.of(NumberOfGuesses);
     }
+	public String EnterGuess(){
+		System.out.println("enter your 5 letter guess");
+		return input.nextLine();
+}
+
+	public String GuessData(String guess){
+		int guessLength = guess.length();
+		if (guessLength != 5 || !guess.matches("[a-zA-Z0-9]{5}")){
```

Comment:
Why are numerals included in input validation? The Acceptance Criteria for #25 indicates that only "alphabetic" inputs are accepted and more specifically: "reject inputs that contain numbers".